src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,27 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <UserInfo user={user} />
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+      {comments.length === 0 ? (
+        <b data-cy="NoCommentsMessage">No comments yet</b>)
+        : <CommentList comments={comments} />}
```

Comment:
Indeed, style can be improved here

```suggestion
      {
        comments.length === 0 
          ? (<b data-cy="NoCommentsMessage">No comments yet</b>)
          : <CommentList comments={comments} />
      }
```